consensus/istanbul/config.go

```diff
@@ -17,10 +17,19 @@
 package istanbul
 
 import (
+	"fmt"
+
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+const (
+	//MinEpochSize represents minimun block that can conform an epoch
```

Comment:
There is a typo in "minimum", but also "conform" isn't right here.  I'd just simplify it as follows:

```suggestion
	//MinEpochSize represents the minimum permissible epoch size
```